tests/cases/compiler/emitClassExpressionInDeclarationFile.ts

```diff
@@ -0,0 +1,30 @@
+// @declaration: true
+export var simpleExample = class {
+    static getTags() { }
+    tags() { }
```

Comment:
Can you also add private member to this? Are we suppose to see private member in the type literal? If yes we should not be writing type of that private property/method because then it would likely start showing errors (with inaccessible symbol)